sdcm/scylla_bench_thread.py

```diff
@@ -152,44 +137,54 @@ def verify_results(self):
 
         return sb_summary, errors
 
-    def _run_stress_bench(self, node, loader_idx, stress_cmd, node_list):
+    def _run_stress(self, loader, loader_idx, cpu_idx):
+        cpu_options = ""
+
+        if self.stress_num > 1:
+            cpu_options = f'--cpuset-cpus="{cpu_idx}"'
+
+        docker = RemoteDocker(loader, "scylladb/hydra-loaders:scylla-bench-v0.1.8",
```

Comment:
It is duplication of the value that will change from time to time.
Please, read the `docker/scylla-bench/image` file's content and reuse here.

Reply:
This would be in the configuration

So point less to read it from the image file, image file can go away